Removing old versions from docs

abrennan89 opened the pull request:
Fixes #1850

RichieEscarez commented on docs/serving/services/_index.md:
```diff
@@ -0,0 +1,8 @@
+type: "docs"
+---
+
+{{% readfile file="README.md" relative="true" markdown="true" %}}
```
This link pulls in a "README.md" file from this `/services/` directory (and no longer requires 'relative' and 'markdown' flags). We do this to hide the README filenames from the URL (bad SEO).
```suggestion
{{% readfile file="README.md" %}}
```

If there is not a README file, we should remove this line. 

Alternatively, you also have the option to add an overview statement in the README.md file to define this section. (we continue to use READMEs to cater to our "GitHub users" (those  who choose to view the source files from GitHub), since the READMEs render by default when you click a directory in GitHub).

RichieEscarez commented on docs/install/Knative-with-MicroK8s.md:
````diff
@@ -30,9 +30,7 @@ sudo snap alias microk8s.kubectl kubectl
 ```shell
 echo 'N;' | microk8s.enable knative
 ```
-This command will install Knative Serving version 0.7.1 and Eventing version 0.7.1 components.
````
This might need to be validated. Its not clear to me if MicroK8s does this, but I do know some packages can remain on back versions.

abrennan89 replied:
Managed to find this conversation: https://github.com/knative/docs/pull/1747#discussion_r325378124
I've reached out to @ammarn911

RichieEscarez replied:
Should we move this into a separate PR to avoid blocking all the other changes here?

abrennan89 replied:
Yeah sounds good to me :slightly_smiling_face: do you want to raise a minor issue for this so we remember? :smile:

RichieEscarez replied:
That would be great, and then you can assign it to @ammarn911. Thanks @abrennan89 !

ammarn911 replied:
@abrennan89 the current versions are 0.9.0

abrennan89 replied:
Thanks @ammarn911 ! @RichieEscarez I'll update this to 0.9.0 then and we can merge?

abrennan89 commented:
Hi @RichieEscarez. The OpenShift docs were removed some time ago when I replaced them with a link to our product docs. For some reason this file was hanging around but it was blank, so I just deleted it? Definitely strange though.
Is there someone who can confirm the MicroK8s thing?

RichieEscarez commented:
> Is there someone who can confirm the MicroK8s thing?
I would try finding the file's author through it's commit history.
